metabrainz/templates/index/gdpr.html

```diff
@@ -15,7 +15,7 @@ <h2>{{ _('Introduction') }}</h2>
   </p>
   <p>
     The good news is that this regulation is roughly in line with our established practices: We’ve always held private information 
-    in a high regard and applied the sort of rules to ourselves as we wish to have our own private data treated. Luckily, this 
+    in a high regard and applied the sort of rules to ourselves as we wish to have our private data treated. Luckily, this 
```

Comment:
"own" here makes the sentence be about the humans behind MetaBrainz rather than the foundation. I wouldn’t remove this.